### PR TITLE
Dilpreet | Stage | Implement material on voucher page (except PO)

### PR DESCRIPTION
**Pull request by dilpreetsinghofficial**
https://app.clickup.com/t/2nvrmj4

### CODE REVIEW

File: apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html [hunk 1]
```diff
@@ -152,7 +153,53 @@ <h4 class="modal-title bg pull-left">{{ localeData?.page_heading }}</h4>
...
+                                                                    [currencyCode]="false"
+                                                                >
+                                                                </amount-field>
+                                                            </span>
+                                                        </p>
+                                                    </div>
+                                                </a>
+                                            </ng-template>
+                                        </select-field>
+                                        <!-- <sh-select
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/advance-receipt-adjustment/advance-receipt-adjustment.component.html]: remove commented code

File: apps/web-giddh/src/app/shared/aside-menu-other-taxes/aside-menu-other-taxes.html [hunk 2]
```diff
@@ -20,7 +20,7 @@ <h3 class="aside-title">{{commonLocaleData?.app_tax_applicable}}</h3>
                         <label>{{commonLocaleData?.app_select_tax}}<span class="text-danger">*</span></label>
 
                         <div class="mr-b2">
-                            <sh-select
+                            <!-- <sh-select
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/aside-menu-other-taxes/aside-menu-other-taxes.html:35]: remove commented code

File: apps/web-giddh/src/app/shared/aside-menu-other-taxes/aside-menu-other-taxes.html [hunk 3]
```diff
@@ -44,12 +55,19 @@ <h3 class="aside-title">{{commonLocaleData?.app_tax_applicable}}</h3>
...
                     <div class="mb-2">
-                        <sh-select
+                        <select-field
+                            [cssClass]="'form-control mat-field-border'"
+                            [placeholder]="commonLocaleData?.app_select_method"
+                            [options]="calculationMethodOptions"
+                            [defaultValue]="defaultOtherTaxesModal.tcsCalculationMethod"
+                        >
+                        </select-field>
+                        <!-- <sh-select
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/aside-menu-other-taxes/aside-menu-other-taxes.html]: remove commented code

File: apps/web-giddh/src/app/shared/aside-menu-recurring-entry/aside.menu.recurringEntry.component.ts [hunk 4]
```diff
@@ -192,4 +192,8 @@ export class AsideMenuRecurringEntryComponent implements OnInit, OnChanges, OnDe
             ];
         }
     }
+    public onIntervalChange(event:any):void{
+console.log(event);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/shared/aside-menu-recurring-entry/aside.menu.recurringEntry.component.ts]: remove console and blank function

File: apps/web-giddh/src/app/voucher/components/voucher-gst-treatment/voucher-gst-treatment.component.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    @Output() public closeEvent: EventEmitter<boolean> = new EventEmitter();
+    /* This will hold local JSON data */
+    @Input() public localeData: any = {};
+    /* This will hold common JSON data */
+    @Input() public commonLocaleData: any = {};
+
+    constructor() {
+    }
+
+    public clickInside(event) {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/voucher/components/voucher-gst-treatment/voucher-gst-treatment.component.ts:26]: add js doc

File: apps/web-giddh/src/app/voucher/components/voucher-print-inplace/voucher-print-in-place.component.ts [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+        private _toasty: ToasterService,
+        private _proformaService: ProformaService,
+        private _receiptService: ReceiptService,
+        private domSanitizer: DomSanitizer,
+        private generalService: GeneralService,
+        private commonService: CommonService,
+        private changeDetection: ChangeDetectorRef
+    ) { }
+
+    public ngOnInit(): void {
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/voucher/components/voucher-print-inplace/voucher-print-in-place.component.ts:48]: add js doc in all functions

File: package.json [hunk 7]
```diff
@@ -202,7 +203,7 @@
     "protractor": "7.0.0",
     "ts-node": "10.9.1",
     "tslint": "6.1.3",
-    "typescript": "^4.6.2",
+    "typescript": "4.5.3",
```
Comment (ravinderwalkover) [inline package.json]: revert this change

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 8]
```diff
@@ -26,6 +28,7 @@ $custom-typography: mat-typography-config(
     word-break: break-word;
     white-space: nowrap;
 }
+
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:31]: remove all white spaces

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/voucher/voucher.component.scss [hunk 9]
```diff
@@ -0,0 +1,2174 @@
+// mixins
+
+/**
+# margin mixin pass direction and number
+*/
+@mixin marg($d, $num) {
```
Comment (the-prateek-mishra) [inline apps/web-giddh/src/app/voucher/voucher.component.scss:6]: Check this file

**Review by the-prateek-mishra — CHANGES_REQUESTED** [review]
